patches/api/0108-EntityShootBowEvent-consumeArrow-and-getArrowItem-AP.patch

```diff
@@ -42,3 +42,13 @@ index 1a8366f6757431baaba4f3d48abea3cf0ec1f1ad..e4efcd757ca51a4edd396f55c812dbe8
      public EntityShootBowEvent(@NotNull final LivingEntity shooter, @Nullable final ItemStack bow, @Nullable final ItemStack consumable, @NotNull final Entity projectile, @NotNull final EquipmentSlot hand, final float force, final boolean consumeItem) {
          super(shooter);
          this.bow = bow;
+@@ -112,9 +137,7 @@ public class EntityShootBowEvent extends EntityEvent implements Cancellable {
+      * consumed).
+      *
+      * @param consumeItem whether or not to consume the item
+-     * @deprecated not currently functional
+      */
+-    @Deprecated
+     public void setConsumeItem(boolean consumeItem) {
+         this.consumeItem = consumeItem;
+     }
```

Comment:
extra diff

Reply:
what do you mean by extra diff? make it into a new patch?

Reply:
Didn't realize the removal of the deprecation notice & annotation was intentional

Reply:
ah ok